dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -42,27 +42,48 @@ ENV BEAT_SETUID_AS={{ .user }}
 {{- if contains .from "ubi-minimal" }}
 RUN for iter in {1..10}; do microdnf update -y && microdnf install -y shadow-utils jq &&  microdnf clean all && exit_code=0 && break || exit_code=$? && echo "microdnf error: retry $iter in 10s" && sleep 10; done; (exit $exit_code)
 {{- else }}
-# Installing jq needs to be installed after epel-release and cannot be in the same yum install command.
-RUN case $(arch) in aarch64) YUM_FLAGS="-x bind-license";; esac; \
-    for iter in {1..10}; do \
-        yum update -y $YUM_FLAGS && \
-        yum install -y epel-release &&  \
-        yum update -y $YUM_FLAGS && \
-        yum install -y jq && \
-        yum clean all && \
-        exit_code=0 && break || exit_code=$? && echo "yum error: retry $iter in 10s" && sleep 10; \
+
+RUN for iter in {1..10}; do \
+        apt update -y && \
+        apt install --no-install-recommends --yes jq && \
+        apt clean all && \
+        exit_code=0 && break || exit_code=$? && echo "apt error: retry $iter in 10s" && sleep 10; \
     done; \
     (exit $exit_code)
 {{- end }}
 
+## Changes Notes: To migratre from CentOS to Ubuntu, the following changes are needed:
+# atk -> libatk1.0-0
+# gtk -> GTK3 is installed by default REMOVED
+# gdk -> libgdk-pixbuf2.0-dev
+# xrandr -> x11-xserver-utils
+# pango -> libpango-1.0-0
+# libXcomposite -> libxcomposite-dev
+# libXcursor -> libxcursor-dev
+# libXdamage -> libxdamage-dev
+# libXext -> libxext-dev 
+# libXi -> libxi-dev
+# libXtst -> libxtst-dev
+# libXScrnSaver -> libxss-dev
+# libXrandr -> libxrandr-dev
+# GConf2 -> libgconf2-4
+# alsa-lib -> libasound2
+# gtk3 -> GTK3 is installed by default REMOVED
+# ipa-gothic-fonts -> fonts-ipafont-gothic
+# xorg-x11-fonts-100dpi -> xfonts-100dpi
```

Comment:
@jsoriano @andrewkroh Do you remember what requires the xorg dependencies?

Reply:
Blind guess: abrt-xorg.

> The abrt-xorg service collects and processes information about crashes of the X.Org server.

Reply:
That must be a dependency required by Synthetics.

Reply:
Ah, I missed the part below that it is only included in the "complete" image. Good to know it is not in the standard image.

@andrewvc FYI

Reply:
Yep!